src/index.html

```diff
@@ -4,14 +4,176 @@
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input 
+          type="date" 
+          name="birth-date" 
+          id="birth"
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement:
+          <input 
+          type="checkbox" 
+          name="agreement"
+           id="agreement"
+          >
+        </label> 
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="form-field">
+            E-mail:
+          <input 
+          type="email" 
+          name="email"
+          id="email" 
+          placeholder="email@example.com" 
+          required
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input 
+          type="password" 
+          name="password"
+          id="password"
+          minlength="4"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+        Do you love cats?
+          <label for="yes">
+            <input 
+            type="radio" 
+            name="cats"
+            id="yes" 
+            value="yes"
+            >
+      Yes
+          </label>
+     
+          <label for="no">
+            <input 
+            type="radio"
+            name="cats" 
+            id="no" 
+            value="no"
+            >
+      No
+          </label>
+        </div>
+        <label class="form-field">
+          What is your favorite color?
+          <input 
+          type="color" 
+          name="color"
+          id="color"
+          value="black"
+          >
+        </label>
+        <label class="form-field">
+            What time do you go to bed?
+          <input 
+          type="time" 
+          name="bedtime" 
+          id="bedtime"
+          >
+        </label>
+        <label class="form-field">
+            What are your favorite brand of cars?
+          <select 
+          name="car" 
+          id="car" 
+          multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="form-field">
+            How do you rate our work?
+          <input 
+          type="range" 
+          name="rate"
+          id="rate"
+          min="0"
+          max="10"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+        <label class="form-field">
+            Comments:
+          <textarea 
+          name="comments" 
+          id="comments" 
+          cols="12" 
+          rows="2" 
+          maxlength="250"
+          > 
+          </textarea>
+        </label>
+        <label class="form-field">
```

Comment:
```suggestion
        </label>
        
        <label class="form-field">
```
Add newlines between labels to make your code easier to read